README.md

```diff
@@ -1,23 +1,61 @@
 # Cloud Foundry Service Broker for Google Cloud Platform
 
-Depends on [lager](https://github.com/pivotal-golang/lager) and [gorilla/mux](https://github.com/gorilla/mux).
+This is the home of the Cloud Foundry Service Broker for Google Cloud Platform. For a demo of installing and using the broker,
+see [here](https://www.youtube.com/watch?v=8nc4624K91A&list=PLIivdWyY5sqKJ48ycao632rEDuVbFm8yJ&index=3)
 
-Requires Go 1.7 and the associated buildpack.
+## Background
 
-## Examples
+### Service Brokers
 
-See the [examples](https://github.com/GoogleCloudPlatform/gcp-service-broker/tree/master/examples/) folder.
+This product is a [Cloud Foundry Service Broker](https://docs.cloudfoundry.org/services/overview.html). It adheres
+to [v2.8](https://docs.pivotal.io/pivotalcf/1-7/services/api.html) of the Service Broker API.
 
-## Prerequisites
+### Google Cloud Platform (GCP)
 
-### Set up a GCP Project
+GCP is a cloud service provider. In addition to VMs and networking, many other useful services are available. The ones
+available through this Service Broker are:
+
+* [PubSub](https://cloud.google.com/pubsub/)
+* [Cloud Storage](https://cloud.google.com/storage/)
+* [BigQuery](https://cloud.google.com/bigquery/)
+* [CloudSQL](https://cloud.google.com/sql/)
+* [ML APIs](https://cloud.google.com/ml/)
+* [Bigtable](https://cloud.google.com/bigtable/)
+* [Spanner](https://cloud.google.com/spanner/)
+* [Stackdriver Debugger](https://cloud.google.com/debugger/)
+* [Stackdriver Trace](https://cloud.google.com/trace/)
+
+
+## Installation
+
+Requires Go 1.8 and the associated buildpack.
+
+* [Installing as a Pivotal Ops Manager tile](#tile)
```

Comment:
It might make sense to just link to http://docs.pivotal.io/partners/gcp-sb/index.html here, saves a click.